.circleci/config.yml

```diff
@@ -60,6 +60,25 @@ jobs:
           fail_only: true
           failure_message: Nightly auth tests failed!
           webhook: $SLACK_WEBHOOK
+  uptime_monitor_waf:
+    working_directory: ~/repo
+    docker:
+      - image: circleci/buildpack-deps:18.04-browsers
+        auth:
+          username: dockstoretestuser
+          password: $DOCKERHUB_PASSWORD
+    steps:
```

Comment:
As an alternative, you could just make the WAF tests part of the other nightly tests, i.e., don't change this file at all, and just add the WAF to the list of tests run as part of `test-dev-no-auth` in the package.json.

Probably faster instead of setting up a new container and what not. Plus it makes invocation manually for staging/prod more straightforward (less likely to overlook).

On the other hand, UI smoke tests have been a little fragile, and you might want to separate these out as more serious.